notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.                opset_version=11,
</pre><p>The description above mentions opset 14 instead of 11</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>removed accidentally copied message from another notebook</p><p> </p>